plugins/huggingface/newhelm/suts/huggingface_client.py

```diff
@@ -357,6 +357,12 @@ def evaluate(self, raw_request: HuggingFaceRequest) -> HuggingFaceResponse:
             sequences = encoded_input["input_ids"]
             scores = output.logits
         else:
+            generation_config = (
```

Comment:
Since this breaks over multiple lines anyway, I think nested if is clearer:

```
generation_config = None
if self.model.config.pad_token_id is None:
    if self.model.config.eos_token_id is not None:
        generation_config = GenerationConfig(pad_token_id=self.model.config.eos_token_id)
```

Alternatively, it might fit on one line you store `self.model.config` in a local variable.

Reply:
I was unsure not sure what might be the most readable way. Updated and also added a comment. Thank you.